response_operations_ui/templates/respondent.html

```diff
@@ -3,18 +3,22 @@
 {% set page_title = "Respondents" %}
 
 {% block main %}
-
+    
     {% include 'partials/flashed-messages.html' %}
 
+    
+    {% set is_respondent_edit_permission = hasPermission('respondents.edit') %}
```

Comment:
This is still really inefficient. If we're going this way, we should get the permissions once and return a dict, and then just do 

`{% set permissions = getPermissions() %}`
`{% set is_respondent_edit_permission = 'respondents.edit' in permissions %}`

Reply:
Until I discuss this with Dan, let's keep it this way and as a part of the caching/ changes to has_permission() we will modify this.